app/Insightly/Resources/InsightlyOpportunityResource.php

```diff
@@ -82,4 +83,40 @@ public function linkContact(int $opportunityId, int $contactId, ContactType $con
 
         $this->insightlyClient->sendRequest($request);
     }
+
+    /**
+     * @throws ContactCannotBeUnlinked
+     */
+    public function unlinkContact(int $opportunityId, int $contactId): void
+    {
+        $getLinksRequest = new Request(
```

Comment:
This can become a reusable method on the opportunities resource. Because later on, we will also need to have it for projects, depending on the state/stage of an Integration.

Reply:
Good point. Will do!

Reply:
I'll make it a private function for now, as I don't know yet which properties should be exposed and don't want to expose everything yet. We can revisit when we need to.